app/models/order_item.rb

```diff
@@ -1,18 +1,13 @@
 class OrderItem < ActiveRecord::Base
-  belongs_to :book
```

Comment:
I noticed you removed the `belongs_to` and added a `book_title` property. The generated schema still shows order_item having a `book_id`, is that intentional?